### PR TITLE
chore: add cloudbuild setup to publish docker images

### PR DESCRIPTION
**Pull request by mpeddada1**
Cloud Build Trigger for the docker image push is set to:

Screenshot 2023-12-14 at 3 06 16 PM

### CODE REVIEW

**Comment by mpeddada1** [comment]
\gcbrun

**Comment by mpeddada1** [comment]
Now running into: 
```
Step #1: 00:00:45:335 [INFO] -------------------------------------------------------
Step #1: 00:00:47:331 [INFO] Running com.google.cloud.kms.it.ITKmsTest
Step #1: 00:00:50:377 [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 3.019 s <<< FAILURE! -- in com.google.cloud.kms.it.ITKmsTest
Step #1: 00:00:50:378 [ERROR] com.google.cloud.kms.it.ITKmsTest.ensureKmsKeyRingExists -- Time elapsed: 2.983 s <<< ERROR!
Step #1: io.grpc.StatusRuntimeException: PERMISSION_DENIED: Permission 'cloudkms.keyRings.get' denied on resource 'projects/cloud-java-ci-test/locations/us/keyRings/gcs_test_kms_key_ring' (or it may not exist).
```

File: .cloudbuild/cloudbuild-test-a.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+timeout: 7200s # 2 hours
+substitutions:
+  _SHARED_DEPENDENCIES_VERSION: '3.20.0' # {x-version-update:google-cloud-shared-dependencies:current}
+  _JAVA_SHARED_CONFIG_VERSION: '1.7.1'
```
Comment (burkedavison) [inline .cloudbuild/cloudbuild-test-a.yaml:18]: Does this need a comment for dependabot to update it? (If so, update in all locations.)

Comment (mpeddada1) [reply .cloudbuild/cloudbuild-test-a.yaml:18]: It's handled by renovate-bot through `renovate.json`.

**Review by burkedavison — APPROVED** [review]

**Pull request merged**